docs/integrations/data-integrations/amazon-s3.mdx

```diff
@@ -20,7 +20,6 @@ WITH
     parameters = {
       "aws_access_key_id": "AQAXEQK89OX07YS34OP"
       "aws_secret_access_key": "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY",
```

Comment:
maybe also to remove values which looks like real and replace them with something like 'YOUR ACCESS KEY',

Reply:
@ea-rus It looks like the documentation for our other integration also use examples that are fake but look real such as [this](https://docs.mindsdb.com/integrations/data-integrations/snowflake). 
@martyna-mindsdb What should our approach be here?

Reply:
What we do in the Minds docs is:
- we provide sample (fake) values for users' reference, as it used to happen in the past that users provided the wrong value because they were not clear on what should be the sample format.
- we note that these must be replaced with real values like `Note that sample parameter values are provided here for reference, and you should replace them with your connection parameters.`

Ref - https://docs.mdb.ai/docs/data_sources

Reply:
@martyna-mindsdb I have added this in as a note.